gp-includes/routes/project.php

```diff
@@ -386,8 +390,11 @@ public function permissions_post( $project_path ) {
 				'locale_slug' => gp_post( 'locale' ),
 				'set_slug'    => gp_post( 'set-slug' ),
 			) );
-			if ( $this->invalid_and_redirect( $new_permission, gp_url_current() ) ) return;
-			$permission = GP::$validator_permission->create( $new_permission );
+			if ( $this->invalid_and_redirect(
+				$new_permission, gp_url_current() ) ) {
+					return;
+			}
```

Comment:
```suggestion
			if ( $this->invalid_and_redirect( $new_permission, gp_url_current() ) ) {
				return;
			}
```

No line break before `$new_permission`.